tools/xplatRegenBytecode.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+#-------------------------------------------------------------------------------------------------------
+# Copyright (C) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+#-------------------------------------------------------------------------------------------------------
+
+# Regenerate embedded bytecode headers.
+# NOTE this script is for linux and macOS only, on windows please use RegenAllByteCode.cmd AND update_bytecode_version.ps1
+# Also note - this script uses paths relative to the tools directory it's in so cd to it before running
+# ch is used to generate bytecode.
+# ch (NoJIT variety) is used to generate NoJIT bytecodes.
+
+
+import subprocess
+import sys
+import uuid
+
+# Compile ChakraCore both noJit and Jit variants
+print("Compiling ChakraCore with no Jit")
+noJitBuild = subprocess.Popen(['../build.sh', '--no-jit', '--test-build', '--target-path=../out/noJit', '-j=2'])
+noJitBuild.wait()
+
+if noJitBuild.returncode!=0:
+    sys.exit("No Jit build failed - aborting bytecode generation")
+
+print("Compiling ChakraCore with Jit")
+jitBuild = subprocess.Popen(['../build.sh', '--test-build', '--target-path=../out', '-j=2'])
+jitBuild.wait()
+
+if jitBuild.returncode!=0:
+    sys.exit("Jit build failed - aborting bytecode generation")
+
+#Regenerate the bytecode
+
+#INTL
+print("Generating INTL bytecode")
+noJitIntlHeader = open('../lib/Runtime/Library/InJavascript/Intl.js.nojit.bc.h','w')
+noJitIntl = subprocess.Popen(['../out/noJit/test/ch', '-GenerateLibraryByteCodeHeader', '-Intl', '../lib/Runtime/Library/InJavascript/Intl.js'],stdout=noJitIntlHeader)
+
+jitIntlHeader = open('../lib/Runtime/Library/InJavascript/Intl.js.bc.h','w')
+jitIntl = subprocess.Popen(['../out/test/ch', '-GenerateLibraryByteCodeHeader', '-Intl', '../lib/Runtime/Library/InJavascript/Intl.js'],stdout=jitIntlHeader)
+
+noJitIntl.wait()
```

Comment:
https://docs.python.org/2/library/subprocess.html#subprocess.check_call might make this process code a bit simpler, since you wont need to wait() and check `returncode` on each call.